app/controllers/candidates/registrations/placement_preferences_controller.rb

```diff
@@ -42,10 +43,21 @@ def set_school
         @school = Candidates::School.find(params[:school_id])
       end
 
+      def set_placement_dates
+        if @school.fixed_dates?
+          @placement_dates = @school
+            .school_profile
+            .bookings_placement_dates
+            .available
+        end
+      end
+
       def placement_preference_params
-        params.require(:candidates_registrations_placement_preference).permit \
+        params.require(:candidates_registrations_placement_preference).permit(
           :availability,
-          :objectives
+          :objectives,
+          :bookings_placement_date_id
+        ).merge(urn: current_urn)
```

Comment:
refactoring the candidate wizard so we don't have to do this merge is on my todo list!